test/Ocelot.UnitTests/Request/Mapper/RequestMapperTests.cs

```diff
@@ -113,11 +114,51 @@ public void Should_handle_no_headers()
     public void Should_map_content()
     {
         this.Given(_ => GivenTheInputRequestHasContent("This is my content"))
-            .And(_ => GivenTheInputRequestHasMethod("GET"))
+            .And(_ => GivenTheInputRequestHasMethod("POST"))
```

Comment:
Attention!
`GET` was replaced with `POST`!!!
Better to convert to `Theory`

Reply:
Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1972/commits/dcbbab7119180345afaf001fbf3664f64b2fafd7 ✔️